api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -243,15 +243,26 @@ function parsePolkadotTransaction(
   }
 }
 
-function getExtrinsicSuccess(extrinsicIndex, blockEvents) {
-  return blockEvents.find(
-    ({ event, phase }) =>
-      parseInt(phase.toHuman().ApplyExtrinsic) === extrinsicIndex &&
-      event.section === `system` &&
-      event.method === `ExtrinsicSuccess`
-  )
-    ? true
-    : false
+function getExtrinsicSuccess(extrinsicIndex, blockEvents, isBatch) {
+  const events = blockEvents
+  .filter(({ phase }) => {
+    parseInt(phase.toHuman().ApplyExtrinsic) === extrinsicIndex // index is a string
+  })
+  // if tx is a batch, we need to check if all of the batched txs went through
+  if (isBatch) {
+    return !!events
+      .find(
+        ({ event }) =>
+          event.section === `utility` &&
+          event.method === `BatchCompleted`
```

Comment:
We should invert here and directly look for the errors so we can get the error codes

Reply:
mmm, something seems to be failing here

Reply:
![image](https://user-images.githubusercontent.com/40721795/88826413-04f4b880-d1c9-11ea-9802-904f5f77aa74.png)

Reply:
Fixed